pkg/workflow/v1/testData/valid.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: This is a workflow
+parameters:
+  hi:
+    default: 5
+    description: "Hello"
+steps:
+  - name: step-1
+    image: image-1
+    
```

Comment:
I think we'll want to bring in a whole bunch more test fixtures here. Let's chat tomorrow about where we can source them. I know the API has a few, and we can probably nab some from https://github.com/puppetlabs/nebula-workflow-examples, the deployment code, and whatever test ones we've run through staging.

Reply:
You're thinking pulling in real workflow files in-use so we can validate that we're not breaking things?

Reply:
No, just copying them in their current state (with some redactions for anything internally sensitive) so we have a corpus of real data to work with.